packages/experimental/Drawer/src/Drawer.tsx

```diff
@@ -0,0 +1,68 @@
+/** @jsx withSlots */
+import { Animated, Modal, TouchableWithoutFeedback, View, StyleSheet } from 'react-native';
+
+import type { UseSlots } from '@fluentui-react-native/framework';
+import { compose, mergeProps, withSlots } from '@fluentui-react-native/framework';
+
+import { stylingSettings } from './Drawer.styling';
+import type { DrawerType, DrawerProps } from './Drawer.types';
+import { DrawerName } from './Drawer.types';
+import { useDrawer } from './useDrawer';
+
+export const Drawer = compose<DrawerType>({
+  displayName: DrawerName,
+  ...stylingSettings,
+  slots: {
+    modal: Modal,
+    backdrop: TouchableWithoutFeedback,
+    backdropContent: Animated.View,
+    content: Animated.View,
+    handle: View,
+  },
+  useRender: (userProps: DrawerProps, useSlots: UseSlots<DrawerType>) => {
+    const Drawer = useDrawer(userProps);
+    const Slots = useSlots(userProps);
+
+    return (final: DrawerProps, children: React.ReactNode) => {
+      const { visible, onClose, onBackdropClick, animationConfig, position, ...rest } = mergeProps(Drawer.props, final);
+      return (
+        <Slots.modal
+          {...rest}
+          visible={visible}
+          onRequestClose={onClose}
+          supportedOrientations={['portrait', 'portrait-upside-down', 'landscape', 'landscape-left', 'landscape-right']}
+          animationType="none"
+          transparent
+        >
+          <Slots.backdrop onPress={onBackdropClick}>
```

Comment:
More for spec review query - duplicated from it.. could you more description on what these slots would handle each

Reply:
Will add description to spec + code.

Reply:
Description as added to the SPEC.md. 
I have already added them to spec doc and will be copied same to SPEC.md when we check in here.

Reply:
Sounds good.